--- pykml/factory.py
+++ pykml-py3/factory.py
@@ -213,4 +213,4 @@
                 pass #variable was not defined
             else:
                 f.close
-    print write_python_script_for_kml_document(doc)
\ No newline at end of file
+    print(write_python_script_for_kml_document(doc))
\ No newline at end of file
--- pykml/parser.py
+++ pykml-py3/parser.py
@@ -5,7 +5,10 @@
 '''
 import sys
 import os
-import urllib2
+try:
+    from urllib2 import urlopen
+except:
+    from urllib.request import urlopen
 from lxml import etree, objectify
 
 OGCKML_SCHEMA = 'http://schemas.opengis.net/kml/2.2.0/ogckml22.xsd'
@@ -21,7 +24,7 @@
                 self.schema = etree.XMLSchema(file=f)
         except:
             # try to open a remote URL
-            f = urllib2.urlopen(schema)
+            f = urlopen(schema)
             self.schema = etree.XMLSchema(file=f)
     
     def validate(self, doc):
@@ -91,7 +94,7 @@
         fileobject = open(uri)
     except IOError:
         try:
-            fileobject = urllib2.urlopen(uri)
+            fileobject = urlopen(uri)
         except ValueError:
             raise ValueError('Unable to load URI {0}'.format(uri))
     except:
